docs/supported_apis/dataframe_supported.rst

```diff
@@ -103,15 +111,16 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``dot``                    | `dot`_                    | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``drop``                   | `drop`_                   | Y                      |                                                    |
+| ``drop``                   | `drop`_                   | Y                      | **Omnisci**:``P``, ``D`` for row drop              |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``droplevel``              | `droplevel`_              | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``drop_duplicates``        | `drop_duplicates`_        | D                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``dropna``                 | `dropna`_                 | Y                      |                                                    |
+| ``dropna``                 | `dropna`_                 | Y                      | **Omnisci**:``P``, ``thresh`` and ``axis`` params  |
```

Comment:
```suggestion
| ``dropna``                 | `dropna`_                 | Y                      | **Omnisci**: ``P``. ``thresh`` and ``axis`` params  |
```

Reply:
Doesn't it seem to be confusing? To me, it does.

Reply:
Ok, how do you read this?

Reply:
For example, someone may think of `P`/`D` as a parameter of the function.